src/toGamut.js

```diff
@@ -193,9 +193,14 @@ export function toGamutCSS (origin, { space = origin.space }) {
 		else if (!inGamut(current, space)) {
 			const clipped = clip(current);
 			const E = deltaEOK(clipped, current);
-			// Note- this is missing several steps of the CSS Gamut Mapping
-			// Algorithm at this point.
 			if (E < JND) {
+				// Note- this implementation does not conform to the CSS Spec.
+				// This removes the check that (JND - E < ε) and its else
+				// statement. For some colors, JND - E was never less than ε, so
+				// the algorithm continued projecting chroma higher, until the
+				// while statement ended. At that point, the chroma was returned
+				// to `max`, which is the original chroma, so the color was
+				// still out of gamut.
```

Comment:
This comment can now be simplified because

 - https://github.com/w3c/csswg-drafts/commit/06748226bb73a7fee401ba0d00c4e4655ec17555

Reply:
This comment is removed now.